x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DatafeedJobsIT.java

```diff
@@ -405,10 +415,16 @@ public void testStopAndRestartCompositeDatafeed() throws Exception {
 
         List<Bucket> scrollBuckets = getBuckets(scrollJobId);
         List<Bucket> compositeBuckets = getBuckets(compositeJobId);
+        assertThat(
+            "composite bucket size " + scrollBuckets + " does not equal scroll bucket size" + compositeBuckets,
```

Comment:
The order is wrong I think. `scrollBuckets` and `compositeBuckets` should be swapped.